blocks-config/uagb-controls/block-icons.js

```diff
@@ -2,7 +2,9 @@
  * Block Icons
  */
 
-const el = wp.element.createElement;
+ import {
+	createElement as el
+} from 'react';
```

Comment:
Can we do something like this?

```suggestion
 import {
	createElement as el
} from '@wordpress/element';
```

Reply:
Resolved.